pkg/authorization/authorizer/authorizer.go

```diff
@@ -231,7 +235,9 @@ func (a openshiftAuthorizationAttributes) ruleMatches(rule authorizationapi.Poli
 
 	if a.verbMatches(util.NewStringSet(rule.Verbs...)) {
 		if a.resourceMatches(resourceNames) {
```

Comment:
can we rename the `resourceNames` variable to `resourceKinds` or `resourceTypes` (I still am not sure which is actually correct) to keep us from losing our minds, now that we actually have resource names?